test/tools_test.cc

```diff
@@ -113,6 +113,34 @@ GTEST_TEST(CreateRoadNetwork, DragwayRoadNetwork) {
   EXPECT_NE(nullptr, dynamic_cast<const dragway::RoadGeometry*>(dut->road_geometry()));
 }
 
+class CreateMaliputOsmRoadNetworkTest : public ::testing::Test {
+ public:
+  static constexpr int kOverwrite{1};
+  static constexpr char MULTILANE_RESOURCE_ROOT[] = "MULTILANE_RESOURCE_ROOT";
```

Comment:
I think you want OSM_RESOURCE_ROOT, right?

I wonder why "MULTILANE_RESOURCE_ROOT" worked here.

Reply:
Good catch. Actually it wasn't being used. Addressed at [b18386e](https://github.com/maliput/maliput_integration/pull/124/commits/b18386e56823b9aac2129c87249352bd464ad279)